src/components/Activity/activity-types.js

```diff
@@ -10,134 +9,77 @@ import iconDraftJury from './assets/activity-icon-draft-jury.svg'
-        description: radspec[actions.ExecuteRuling](disputeId),
-      }
-    },
-    [actions.Heartbeat]({ transitions }) {
-      return {
-        title: 'Update term',
-        icon: iconCourtLogo,
-        description: radspec[actions.Heartbeat](transitions),
-      }
-    },
-    [actions.LeakVote]({ voteId, voter }) {
-      return {
-        title: 'Leak vote',
-        icon: iconCommitVote,
-        description: radspec[actions.LeakVote](voteId, voter),
-      }
-    },
-    [actions.RevealVote]({ roundId, disputeId }) {
-      return {
-        title: 'Reveal vote',
-        icon: iconAnj,
-        description: radspec[actions.RevealVote](roundId, disputeId),
-      }
-    },
-    [actions.SettleReward]({ roundId, disputeId }) {
-      return {
-        icon: iconAnj,
-        title: 'Settle reward',
-        description: radspec[actions.SettleReward](roundId, disputeId),
-      }
-    },
-    [actions.SettleAppealDeposit]({ roundId, disputeId }) {
-      return {
-        icon: iconAnj,
-        title: 'Settle appeal deposit',
-        description: radspec[actions.SettleAppealDeposit](roundId, disputeId),
-      }
-    },
-    transaction({ transactionHash }) {
-      return {
-        title: 'Transaction',
-        icon: iconAnj,
-        description: `
-          Transaction: ${transactionHash}
-        `,
-      }
-    },
-    [actions.WithdrawAnj]({ amount }) {
-      return {
-        icon: iconAnj,
-        title: 'Withdraw ANJ',
-        description: radspec[actions.WithdrawAnj](amount),
-      }
-    },
-  })
-)
+const ACTIVITY_TYPES = {
+  [actions.APPROVE_FEE_DEPOSIT]: {
+    icon: iconAnj,
+    title: 'Approve fee deposit',
+  },
+  [actions.ACTIVATE_ANJ]: {
+    icon: iconAnj,
+    title: 'Activate ANJ',
+  },
+  [actions.APPEAL_RULING]: {
+    title: 'Appeal ruling',
+    icon: iconAppealRuling,
+  },
+  [actions.CLAIM_REWARDS]: {
+    title: 'Claim rewards',
+    icon: iconClaimRewards,
+  },
+  [actions.CLAIM_SUBSCRIPTION_FEES]: {
+    title: 'Claim Subscription rewards',
+    icon: iconClaimRewards,
+  },
+  [actions.COMMIT_VOTE]: {
+    title: 'Commit vote',
+    icon: iconCommitVote,
+  },
+  [actions.CONFIRM_APPEAL]: {
+    title: 'Confirm appeal',
+    icon: iconAppealRuling,
+  },
+  [actions.DEACTIVATE_ANJ]: {
+    icon: iconAnj,
+    title: 'Deactivate ANJ',
+  },
+  [actions.DRAFT_JURY]: {
+    title: 'Draft jury',
+    icon: iconDraftJury,
+  },
+  [actions.EXECUTE_RULING]: {
+    title: 'Execute ruling',
+    icon: iconExecuteRuling,
+  },
+  [actions.HEARTBEAT]: {
+    title: 'Update term',
+    icon: iconCourtLogo,
+  },
+  [actions.LEAK_VOTE]: {
+    title: 'Leak vote',
+    icon: iconCommitVote,
+  },
+  [actions.REVEAL_VOTE]: {
+    title: 'Reveal vote',
+    icon: iconAnj,
+  },
+  [actions.SETTLE_REWARD]: {
+    icon: iconAnj,
+    title: 'Settle reward',
+  },
+  [actions.SETTLE_APPEAL_DEPOSIT]: {
+    icon: iconAnj,
+    title: 'Settle appeal deposit',
+  },
+  transaction: {
```

Comment:
Just wondering if we should move this to be the last one or etc.

It's kind of weird just sticking out here (and in fact, do we really need this?)

Reply:
Hmm yeah probably we can just remove it, there's no really an action that will use this default activity.